src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Media/Given_BasicAutomatedTransformation.cs

```diff
@@ -0,0 +1,241 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NUnit.Framework;
+using SamplesApp.UITests;
+using SamplesApp.UITests.TestFramework;
+using Uno.UI.RuntimeTests.Helpers;
+using Uno.UITest;
+using Windows.Foundation.Metadata;
+using Windows.UI.Tests;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Imaging;
+using Windows.UI.Xaml.Shapes;
+using static Private.Infrastructure.TestServices;
+using FluentAssertions;
+using FluentAssertions.Execution;
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Media
+{
+	[TestClass]
+	[RunsOnUIThread]
+	public class Basics_AutomatedTransformation
+	{
+
+		//Web Assembly does not have a helper that take screenshots yet
+		//MacOs interprets colors differently
+#if !__WASM__ && !__MACOS__
+
+		private const string NoColor = "#00000000";
```

Comment:
https://github.com/unoplatform/uno/blob/4.4.5/src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Media/Transform_Tests/Basics_Automated.cs#L214
original code uses `Color.White`; `#00000000`, transparent black, isnt exactly the closest